naga/tests/in/atomicTexture.param.ron

```diff
@@ -0,0 +1,24 @@
+(
+	god_mode: true,
+	spv: (
+		version: (1, 0),
+		capabilities: [],
+	),
+	hlsl: (
+		shader_model: V6_6,
```

Comment:
Does this actually need SM6_6?

Reply:
I don't think it is, 6.6 added 64 bit atomic overloads (https://github.com/gfx-rs/wgpu/pull/5537).